### ISSUE TITLE
JSONDecodeError : Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by FedericoSala** [denisneuf/python-amazon-ad-api#58]
**Error message**
JSONDecodeError                           Traceback (most recent call last)
<ipython-input-14-c5f77c62acba> in <module>
----> 1 results = sponsored_products.Campaigns(credentials=my_credentials).list_campaigns()

/opt/anaconda3/lib/python3.7/site-packages/ad_api/base/helpers.py in wrapper(*args, **kwargs)
     19         def wrapper(*args, **kwargs):
     20             kwargs.update({"path": path, "method": method})
---> 21             return function(*args, **kwargs)
     22         wrapper.__doc__ = function.__doc__
     23         return wrapper

/opt/anaconda3/lib/python3.7/site-packages/ad_api/api/sp/campaigns.py in list_campaigns(self, **kwargs)
    334 
    335         """
--> 336         return self._request(kwargs.pop('path'),  params=kwargs)
    337 
    338     @sp_endpoint('/v2/sp/campaigns/{}', method='GET')

/opt/anaconda3/lib/python3.7/site-packages/ad_api/base/client.py in _request(self, path, data, params, headers)
    269                 logging.info(data)
    270 
--> 271         return self._check_response(res)
    272 
    273     # @staticmethod

/opt/anaconda3/lib/python3.7/site-packages/ad_api/base/client.py in _check_response(self, res)
    291             raise exception(dictionary)
    292 
--> 293         data = json.loads(str_content)
    294 
    295         if type(data) is dict and data.get('code') == 'UNAUTHORIZED':

/opt/anaconda3/lib/python3.7/json/__init__.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    346             parse_int is None and parse_float is None and
    347             parse_constant is None and object_pairs_hook is None and not kw):
--> 348         return _default_decoder.decode(s)
    349     if cls is None:
    350         cls = JSONDecoder

/opt/anaconda3/lib/python3.7/json/decoder.py in decode(self, s, _w)
    335 
    336         """
--> 337         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    338         end = _w(s, end).end()
    339         if end != len(s):

/opt/anaconda3/lib/python3.7/json/decoder.py in raw_decode(self, s, idx)
    353             obj, end = self.scan_once(s, idx)
    354         except StopIteration as err:
--> 355             raise JSONDecodeError("Expecting value", s, err.value) from None
    356         return obj, end
JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**To Reproduce**
1. from ad_api.api import sponsored_products
2. my_credentials = dict(
refresh_token = "xxxx"
client_id = "xxxx"
client_secret = "xxxx"
profile_id = "xxxxx"
)
3. results = sponsored_products.Campaigns(credentials=my_credentials).list_campaigns()

**Comment by FedericoSala** [comment]
Hi Daniel,
I didn't use the commas in the issue but I used them in the code, my bad.
Anyway, I tried with the new credentials dict but I got the same error, here's a snippet of my code, am I doing anything wrong?
Is it possible I have this error because there are no active campaigns for these credentials?

`from ad_api.api import sponsored_products
from ad_api.base import AdvertisingApiException


my_credentials = dict(
refresh_token=REFRESH_TOKEN,
client_id=CLIENT_ID,
client_secret=CLIENT_SECRET,
profile_id=PROFILE_ID
)


try:
    results = sponsored_products.Campaigns(credentials=my_credentials).list_campaigns()

except AdvertisingApiException as error:
    print(error)`
